qa/tasks/cephfs/filesystem.py

```diff
@@ -1440,3 +1489,44 @@ def data_scan(self, args, quiet=False, worker_count=1):
 
     def is_full(self):
         return self.is_pool_full(self.get_data_pool_name())
+
+    def grow(self, new_max_mds):
```

Comment:
In other tests we don't really go to all of this trouble. Instead just do:

````
self.fs.set_max_mds(foo)
status = self.fs.wait_for_daemons()
````

There's not much reason to keep any of these extra checks.